src/components/TransactionDetails/AddLiquidityTransactionDetails.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <component :is="infoOnly ? 'div' : 'transaction-details'" class="info-line-container">
```

Comment:
Looks like this condition could be used inside `TransactionDetails` component - if props `infoOnly` is true, render `div` tag, else render current `collapse` structure. (Just for remove duplicated conditions in every details component)

Reply:
That will cause "Maximum call stack size exceeded" range error since in our AddLiqTransactionDetails we have to put <transaction-details info-only="false"> but inside TransactionDetails itself we would write again <component :is="infoOnly ? 'div' : 'transaction-details'" so that basically means rendering itself again inside <transaction-details> component

Also it relies on slots, so the decision what to pass inside component must be done outer